apps/prairielearn/elements/pl-big-o-input/pl-big-o-input.py

```diff
@@ -70,11 +70,13 @@ def prepare(element_html: str, data: pl.QuestionData) -> None:
     )
 
     if len(variables) > 1:
-        raise ValueError(f"Only one variable is supported for question {name}")
+        msg = f"Only one variable is supported for question {name}"
+        raise ValueError(msg)
```

Comment:
🤮 I can't say I love this (courtesy of https://docs.astral.sh/ruff/rules/f-string-in-exception/)

Reply:
If you feel very strongly about using this rule, please split this specific rule into another PR, the sheer number of changes from this one rule make it hard to review everything else.

Reply:
This is a rather opinionated change, so I would be fine reverting that.

Reply:
Agree that this is a little excessive...

Reply:
Reverted in [70e596d](https://github.com/PrairieLearn/PrairieLearn/pull/11104/commits/70e596dbc3f315a239a0409d460795360d7e8120) and [33920e5](https://github.com/PrairieLearn/PrairieLearn/pull/11104/commits/33920e523b0d3c240e787fcd62d8de907ca545fb).